frontend/src/components/Header/Header.css

```diff
@@ -1,38 +1,121 @@
 .header-container {
-  display: flex;
-  justify-content: space-between;
-  background: hsl(0, 0%, 77%);
-  align-items: center;
+	width: 100%;
+	white-space: nowrap;
 }
 
 .org-name {
-  font-size: 20px;
+	font-size: 20px;
 }
 
 .nav-links {
-  list-style: none;
-  display: flex;
-  justify-content: flex-end;
-  align-items: center;
-  margin: 0;
+	list-style: none;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	margin: 0;
+}
+
+.nav-linkb {
+	padding: 20px;
+	color: black;
+	height: 100%;
 }
 
 .nav-link {
-  padding: 20px;
-  color: hsl(101,43%,42%);
-  height: 100%;
+	padding: 20px;
```

Comment:
I mentioned this in @Aleksandandar-Nedelkovski's PR #107 ; this is going to cause issues. Remove this styling from the `nav-links` class and apply a new class to the elements you want to style.